package/Dockerfile

```diff
@@ -19,34 +19,22 @@
 # Note that fluentd is run with root permssion to allow access to
 # log files with root only access under /var/log/containers/*
 
-FROM debian:stretch-slim
+FROM alpine:3.14
 
-ARG DEBIAN_FRONTEND=noninteractive
-
-COPY clean-apt /usr/bin
-COPY clean-install /usr/bin
 COPY Gemfile /Gemfile
 COPY Gemfile.lock /Gemfile.lock
 
-# 1. Install & configure dependencies.
-# 2. Install fluentd via ruby.
-# 3. Remove build dependencies.
-# 4. Cleanup leftover caches & files.
-# RUN BUILD_DEPS="make gcc g++ libc6-dev ruby-dev libffi-dev" \
-RUN BUILD_DEPS="make ruby-dev build-essential autoconf automake libtool libsnappy-dev" \
-    && clean-install $BUILD_DEPS \
-                     ca-certificates \
-                     libjemalloc1 \
-                     ruby \
+RUN apk update \
+    && apk add --no-cache ca-certificates ruby ruby-dev ruby-irb ruby-etc ruby-webrick build-base \
     && echo 'gem: --no-document' >> /etc/gemrc \
-    && gem install bundle -v 0.0.1 \
+    && gem install bundler \
     && bundle install \
```

Comment:
Would using strict version here prevent random issues in the future?

Reply:
We could do this, but bundler is the bootstrapper for the gems themselves, so leaving it unversioned should be fine. I'd be happy to change it though if it's a non-starter.